data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -16,30 +16,37 @@
 
     [story]
         [part]
-            music=the_city_falls.ogg
+            music=elvish-theme.ogg
             # po: If the target language has a specialized term for a fortified town, like Latin "oppidum", use it for Parthyn.
-            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the baron of the city. Every summer when the mountain passes became clear, orcs came down from the north to prove themselves in battle. Every year Parthyn’s guards repelled the raids, though some of the townsfolk were always lost. Malin’s father led raiding parties to disrupt the orc encampments and send the orcs back to the north."
+            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the baron of the city. Every summer, by the time the mountainous paths and high passes shed their wintry gowns of snow, orcs descended from their northern hills to ravage the settlements at the frontier. Every year, Malin’s father led the townsfolk to repel the raids and force the orcs to retreat back to the north."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            story= _ "When Malin was 10, a mage who passed through the town sensed strong magical abilities in the boy. His parents, seeing a chance for him to live away from the constant battles with the orcs, arranged for him to be sent to the mage community on the Isle of Alduin. There he studied for eight years, while back at home the orc raids increased in their frequency and ferocity. Before Malin could finish his training, however, he was expelled."
+            story= _ "When Malin was 10, a mage who passed through the town sensed strong magical abilities in the boy. His parents, seeing a chance for him to escape the constant war with the orcs, arranged for him to be sent to the mages at the Isle of Alduin. He studied there for eight years, learning all manner of magic, from basic control over the elements, to the mystic arts and scrying, to alchemy and simple medicine."
```

Comment:
New lore, I just added it as extra examples, although I think there are quite related things with Crelanu and Ravanal (LoW and EI, no more spoilers than that for you :p).

Reply:
Constant battles is slightly awkward wording, it would either be "constant battle" or "endless battles". If either of those are fine with you I'll change it to that instead.